### ISSUE TITLE
Update needed for haadcode/ipfs-pubsub-room#orbitdb package

### ISSUE THREAD

**Issue by krahimian** [orbitdb/orbit-db-pubsub#7]
Hey - ipfs-pubsub-room needs to be updated to use the correct method when dialing with protocol

ipfs-shipyard/ipfs-pubsub-room@852d0fa

**Comment by ztl8702** [comment]
+1 for this. I just got hit by a `ipfs-pubsub-room` bug that was fixed in the latest version.

**Comment by ztl8702** [comment]
As a workaround, I added these lines to my `package.json` and it seems to resolve the problem for now:

```json
"resolutions": {
    "orbit-db/orbit-db-pubsub/ipfs-pubsub-room": "1.2.0"
}
```
Reference: https://yarnpkg.com/lang/en/docs/selective-version-resolutions/

I am using `yarn 1.5.1`.

----
However I did noticed that @haadcode maintains a separate version of `ipfs-pubsub-room`, and there are a few patches applied to the `orbitdb` branch: 
- https://github.com/haadcode/ipfs-pubsub-room/commit/2191cb9f8fd4a815b0485e906713c55cc385f05f
- https://github.com/haadcode/ipfs-pubsub-room/commit/68aecffc0c89b2430349d59c2556c0f70b2f96e5

Not sure what effects they have tho. Would love to help if possible.

**Comment by haadcode** [comment]
Thanks for reporting the issue @krahimian and for providing details @ztl8702! This was indeed a problem that popped up with the latest updates to IPFS due *us using a fork*.

This has now been fixed in https://github.com/orbitdb/orbit-db-pubsub/pull/8 and I just published [v0.4.0](http://npmjs.com/package/orbit-db-pubsub) on npm that contains the fix.

Let us know if there are any more problems!
